docs/backend/emails.md

```diff
@@ -2,6 +2,24 @@
 title: Emails
 ---
 
+# Setting up email
+
+If you would like to enable transactional email using services like SendGrid or
+Mailgun, you can configure it by using the following environment variables:
+
+```shell
+SMTP_ADDRESS=         # ie. "smtp.sendgrid.net"
+SMTP_PORT=            # ie. 587
+SMTP_DOMAIN=
+SMTP_USER_NAME=
+SMTP_PASSWORD=
+SMTP_AUTHENTICATION=  # defaults to :plain
+```
+
+We follow the standard `ActionMailer` configuration. For more info, please check out
+Rails'
+[official documentation](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration).
```

Comment:
Huh, I always thought that breaking up lines in Markdown would add line breaks but [apparently not](https://github.com/forem/forem/blob/c25ca98751feab9bdf2fe91e4db7bd2eecef8153/docs/backend/emails.md)? TIL I don't need to put entire paragraphs on a single line! :joy:

Reply:
The magic of prettier:

https://github.com/forem/forem/blob/22eaecba1ff9dac81846e2ef06651394e821e6fe/package.json#L72-L74

:D

Reply:
It really depends on the Markdown processor's setting, so this format could be rendered differently elsewhere